hugo/assets/scss/components/teaser.scss

```diff
@@ -58,13 +59,46 @@
         }
     }
 
+    &:focus,
     &:hover {
         .link__text {
             background-position-x: 0;
             background-size: 100% var(--link-border-width, 2px);
         }
     }
 
+    &--search {
+        border-bottom: 1px solid transparentize($c-blue--darker, 0.8);
+        border-radius: 0;
+
+        em {
+            color: $c-pink;
+            font-weight: $weight-bold;
+        }
+
+        #{ $self }__title {
+            margin: 0;
+        }
+
+        #{ $self }__meta {
+            @include style-text-small;
+        }
+
+        #{ $self }__excerpt {
+            font-style: italic;
+            font-weight: $weight-medium;
+            margin-top: 0.5rem;
+        }
+    }
+
+    @include screen($screen-normal) {
+        &--search {
+            #{ $self }__container {
+                padding: 2rem 20px 20px;
```

Comment:
maybe use the p-gutter variable intead of 20px?

Reply:
done